roboteam_interface_vue/src/modules/composables/ai-controller.ts

```diff
@@ -29,6 +30,43 @@ export const useAiController = defineStore('aiController', () => {
   // Actions
   const setBallPos = (x: number, y: number) => send('setBallPos', { x, y })
 
+  const robotsToSide = ( () => {
+    let teleportRobot = []
+    for(const team of [proto.Team.BLUE, proto.Team.YELLOW]){
+      for (let i = 0; i < 16; i++) {
+        const x = -6 + i * 0.3
+        teleportRobot.push({
+          'id': { 'id': i, 'team': team },
+          'x' : (team === proto.Team.YELLOW ? x : -x),
+          'y' : -4.6,
+          'orientation' : 0
+        })
+      }
+    }
+    return {
+      'control' : {
+        'teleportRobot' : teleportRobot,
+      }
+    }
+  })()
+
+  const ballToCenter = {
+    'control' : {
+      'teleportBall' : {
+        'x' : 0, 'y' : 0, 'z' : 0, 'vx' : 0, 'vy' : 0, 'vz' : 0
+      }
+    }
+  }
+
+  const commandMap = new Map<string, Object>([
+    ['robotsToSide', robotsToSide],
+    ['ballToCenter', ballToCenter],
+  ])
```

Comment:
I think this would be better placed directly in the simulator-widget. I cannot think of another place where would you invoke this and, thus, as is it pollutes the ai-controller

Reply:
Done